sdk/core/core-autorest/api-extractor.json

```
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "mainEntryPointFilePath": "types/latest/src/index.d.ts",
```

Nit: Should this be:
```suggestion
  "mainEntryPointFilePath": "types/src/index.d.ts",
```
I don't think `latest` will exist until the types are rolled up.

---

tsconfig puts them in latest directly: `"declarationDir": "./types/latest"`

---

the downlevel compiler only creates the `types/3.1` folder, it doesn't touch `latest`

---

Ah, I missed that. Nevermind then!